packages/netlify-cms-backend-github/src/API.js

```diff
@@ -692,7 +719,7 @@ export default class API {
     return this.request(`${this.repoURL}/pulls/${prNumber}`, {
       method: 'PATCH',
       body: JSON.stringify({
-        state: closed,
+        state: 'closed',
```

Comment:
Not sure how this worked, but `closed` existing on `window` thus it didn't fail:
![image](https://user-images.githubusercontent.com/26760571/61468645-6bc07e80-a986-11e9-95f2-fbdff3627373.png)

But actually send `state: false` and the REST API ate it and closed the pr